articles/getting-started/install-configure-bonsai.md

```diff
@@ -0,0 +1,86 @@
+---
+uid: BonsaiInstallationAndConfiguration
+title: Bonsai Installation and Configuration
+---
+
+# Bonsai Installation and Configuration
+
+## Install Bonsai
+
+To download Bonsai, select between the portable download and the installer download [here](https://bonsai-rx.org/docs/articles/installation.html).
+
+*   The **Portable** download installs a self-contained sandboxed Bonsai. Portable environments enable users to switch between different environments to prevent package conflicts or confusion between similar packages
+
+    *   To install from the **Portable** download, extract the downloaded file. You can start the portable Bonsai by running the `Bonsai.exe` that is inside the extracted folder  
```

Comment:
Add a line about making a shortcut so it is easier to find the different executables as well.

Reply:
done!